src/library/export/libraryexporter.cpp

```diff
@@ -64,9 +64,7 @@ void LibraryExporter::beginEnginePrimeExport(
             &EnginePrimeExportJob::failed,
             this,
             [](const QString& message) {
-                QMessageBox::critical(nullptr,
-                        tr("Export Failed"),
-                        QString{tr("Export failed: %1")}.arg(message));
+                QMessageBox::critical(nullptr, tr("Export Failed"), message);
```

Comment:
The old formatting just led to a redundant message being presented to the user like "Export failed: Fail to export".  The title and icon of the message box makes it perfectly clear that the export has failed.  This change makes the error message more succinct and relevant to the user.